Sources/PubNubChatComponents/Message List/MessageListViewController.swift

```diff
@@ -82,6 +98,63 @@ open class ChatViewController: CollectionViewComponent {
     })
   }
   
+  open override func onLongPressGestureRecognized(gesture: UIGestureRecognizer) {
+    
+    if gesture.state == .began {
+      
+      // Finds the cell that was pressed
+      guard let affectedCell = collectionView.visibleCells.first(where: {
+        $0.frame.contains(gesture.location(in: collectionView))
+      }) as? MessageListItemCell else {
```

Comment:
I made a casting to `MessageListItemCell` in order to get your `reactionListView` that stores Unicode values I need to populate a picker view. Another reason I need a `reactionListView` is because the `messageActionTapped:` requires from me to pass selected `MessageReactionButtonComponent`.

Do you see a way how to get rid of casting to `MessageListItemCell` and accessing your `reactionListView`?

Reply:
I think in the ideal implementation the Reaction Picker displayed from the long press would be the "source of truth" for which reactions are supported by the application, and the inline message reaction list (from the cell) would just display anything that is passed to it.  I hardcoded the supported reactions because it was the easier option, but eventually that reaction list will need to support dynamic message reactions. 

I believe the initial scope was to the have the list of supported reactions somehow set inside of the configuration for the Message List, and if the list is passed in then that would signal that Message Reactions should be supported.  That  was eventually moved out of scope, and replaced with a simple boolean flag with a hardcoded list. 

I don't think anything needs to be changed now.  We can create JIRAs that would expand the scope of the Message Reactions to support a user specified list of Message Reactions.